_posts/2018-09-19-cockpit-178.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Cockpit 178
+author: mvo
+date: 2018-09-19
+tags: cockpit linux
+slug: cockpit-178
+category: release
+summary: No more KubeVirt
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 178.
+
+### Support for KubeVirt has been removed
+
+We can't unfortunately test and support the KubeVirt related features
+of the Kubernetes page anymore.
+
+KubeVirt support will be part of the OpenShift console.
```

Comment:
This should link to https://lists.fedorahosted.org/archives/list/cockpit-devel@lists.fedorahosted.org/thread/SBDE64DAIVTLIYRGVMESUPR5STRRFCGW/ for a more detailled rationale.

Can we reformulate this to point out that community interest in this feature has waned, as the attention is now on integrating this into the OpenShift console? We only tested it against obsolete kubevirt versions, and it's not even clear whether it works with current versions.

Reply:
My first initial pass was:

> Unfortunately, we are no longer able to test and support features related to KubeVirt on the Kubernetes page.
>
> For people using KubeVirt, we suggest [using this feature on the OpenShift console](https://blog.openshift.com/a-first-look-at-kubevirt/).

I'll rework this with your comments above and do a grammar check on it all.